src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -857,6 +857,30 @@ public void switchFolder(FolderElement switchTo) {
 
 		upFolder.setEnabled(!currentFolder.isRoot());
 	}
+	
+	private void filterElement(ModElementType type) {
+		String filter = "f:" + type.getReadableName().replace(" ", "").toLowerCase(Locale.ENGLISH) + " ";

```

Comment:
You can just call `filter(String text)` here with type.getReadableName().replace(" ", "").toLowerCase(Locale.ENGLISH) so you don't duplicate the code

Reply:
not bad idea